src/app/interfaces/fibre-channel.interface.ts

```diff
@@ -1,9 +1,11 @@
+import { IscsiTarget } from 'app/interfaces/iscsi.interface';
+
 export interface FibreChannelPort {
   id: number;
   port: string;
   wwpn: string | null;
   wwpn_b: string | null;
-  target: unknown; // TODO: Probably IscsiTarget
+  target: IscsiTarget;
```

Comment:
hey Alex
`target` fields has prefix and it is different from IscsiTarget interface, see [here](https://github.com/truenas/webui/pull/11146/files#diff-bd7f460141a992b935ee73bb08a40daed8c278f5239f4ad4e88c19e6c656fa93R9). feel free to keep it as is.

Reply:
thanks Denys, I will try to leave it as `unknown` then

Reply:
@denysbutenko I'll use your interface, since unknown causes issues in code:

```
  target: FibreChannelTarget;
}

export interface FibreChannelTarget {
  id: number;
  iscsi_target_name: string;
  iscsi_target_alias: string | null;
  iscsi_target_mode: string;
  iscsi_target_auth_networks: string[];
  iscsi_target_rel_tgt_id: number;
}
```